index.js

```diff
@@ -1,480 +1,203 @@
-          exports.authorizedTimestamp = false
-          if (cb) cb(body, false)
-        }
-      })
-  }
-}
-/**
-* Requests a single bib data from the API
-*
-* Return format:
-* { data: { total: 1, entries: [ [Object] ] },
-*  url: 'https://nypl-sierra-test.iii.com/iii/sierra-api/v3/bibs/17292415?fields=default,fixedFields,varFields,normTitle,normAuthor,orders,locations' }
-*
-* @param  {string} bibId - the bnumber of the bib you want to request
-* @param  {function} cb - callback
-*/
-exports.requestSingleBib = (bibId, cb) => {
-  if (!exports.authorizedToken) {
-    console.error('No authorizedToken set')
-    if (cb) cb('No authorizedToken set', false)
-  } else {
-    var url = `${exports.credsBase}bibs/${bibId}?fields=default,fixedFields,varFields,normTitle,normAuthor,orders,locations`
-    // use the bearer auth token
-    request.get(url, {
-      'timeout': 120 * 1000,
-      'auth': {
-        'bearer': exports.authorizedToken
-      }
-    },
-      (error, response, body) => {
-        if (error) console.error(error)
-        if (response.statusCode && response.statusCode === 200) {
-          var result = {}
-          result.total = 1
-          result.entries = []
-          result.entries[0] = JSON.parse(body)
-          if (cb) cb(null, {data: result, url: url})
-        } else {
-          if (cb) cb(body, false)
-        }
-      })
-  }
-}
-/**
-* Requests a bib range from the API
-*
-* Return format:
-* { data: { total: 1, entries: [ [Object] ] },
-*  url: 'https://nypl-sierra-test.iii.com/iii/sierra-api/v3/bibs/?limit=1&id=17292415&fields=default,fixedFields,varFields,normTitle,normAuthor,orders,locations' }
-*
-* @param  {string} bibIdStart - the bnumber of the bib you want to request
-* @param  {string} bibIdEnd - the bnumber of the bib you want to request
-* @param  {function} cb - callback
-*/
-exports.requestRangeBib = (bibIdStart, bibIdEnd, cb) => {
-  if (!exports.authorizedToken) {
-    console.error('No authorizedToken set')
-    if (cb) cb('No authorizedToken set', false)
-  } else {
-    var limit = ''
-    if (bibIdEnd === '') limit = '&limit=50'
-    var url = `${exports.credsBase}bibs/?id=[${bibIdStart},${bibIdEnd}]${limit}&fields=default,fixedFields,varFields,normTitle,normAuthor,orders,locations`
-    console.log(url)
-    // use the bearer auth token
-    request.get(url, {
-      'timeout': 120 * 1000,
-      'auth': {
-        'bearer': exports.authorizedToken
-      }
-    },
-      (error, response, body) => {
-        if (error) console.error(error)
-        if (!response) {
-          if (cb) cb(error, false)
-          return false
-        }
-        if (response.statusCode && response.statusCode === 200) {
-          if (cb) cb(null, {data: JSON.parse(body), url: url})
-        } else {
-          if (cb) cb(body, false)
-        }
-      })
+async function authenticate(_retryCount = 1) {
+  try{
+  if (!credsKey || !credsSecret || !baseUrl) {
```

Comment:
This linting doesn't look right. Can we apply standardjs or something close? Indentation isn't quite right and looks like we're sometimes using semicolons, sometimes not.